src/pages/term/Term.tsx

```diff
@@ -1,10 +1,20 @@
+// preact imports
 import { useEffect, useState } from 'preact/hooks';
-import { termsData } from './termsData';
+import { Fragment } from 'preact/jsx-runtime';
 import { route } from 'preact-router';
-import ITerm from '~/models/ITerm';
 import Markdown from 'preact-markdown';
+
+// yeda imports
+import { termsData } from './termsData';
+import ITerm from '~/models/ITerm';
 import useChangeDescription from '../../hooks/useChangeDescription';
 import useChangeTitle from '../../hooks/useChangeTitle';
+import { PrimaryButton } from '~components/UI/Buttons/Buttons';
+import styles from './Term.module.sass';
+
+// svgs
+import backArrow from '../../../public/assets/backArrow.svg';
+import nextArrow from '../../../public/assets/nextArrow.svg';
```

Comment:
The assets of the current components should be in the same folder, it's better for organization, project structure, and clean code. Also, keeping the `public` folder minimal is important, what aren't must be there, shouldn't be there.
```suggestion
import backArrow from './icons/backArrow.svg';
import nextArrow from './icons/nextArrow.svg';
```